waveguide_mode_converter/README.md

```diff
@@ -47,7 +47,6 @@ python3 ./run.py designs/converter_*.csv
 
 Will produce the output below:
 
-```
 # Design file, Length scale (nm), Worst-case reflection (dB), Worst-case transmission (dB)
```

Comment:
This line is _not_ meant to be markdown. It's the header of the CSV file whose contents are displayed in the code block. When copying and importing this CSV as a Pandas data frame, these headers are used to name the columns.

Reply:
Please also keep the code blocks "```"

Reply:
Fixed.